src/views/Dashboard/Hospital/Registration/index.vue

```diff
@@ -160,25 +160,25 @@ export default {
     },
     emailRules() {
       return [
-        val => !!val || 'E-mail is required',
+        val => !!val || 'This field is required',
         val => /.+@.+\..+/.test(val) || 'E-mail must be valid',
```

Comment:
Make the error message consistent with other email field validation :
Email is invalid. It should contain @ followed by a domain